mix.exs

```diff
@@ -159,7 +159,7 @@ defmodule Guardian.Mixfile do
       {:poison, "~> 2.2 or ~> 3.0"},
 
       # Optional dependencies
-      {:phoenix, "~> 1.0 or ~> 1.2 or ~> 1.3", optional: true},
+      {:phoenix, "~> 1.0 or ~> 1.2 or ~> 1.3 or ~> 1.4.0-rc.3", optional: true},
```

Comment:
I've never done this, but is it possible to give maximum flexibility by just changing `... or ~> 1.3` to `... or >= 1.3`?

Reply:
@k-cross yes it is possible to use `>= 1.3`.

Reply:
Might as well make it `>= 1.0` unless you're expecting to make breaking changes.